lib/procore/version.rb

```diff
@@ -1,3 +1,3 @@
 module Procore
-  VERSION = "0.8.6".freeze
+  VERSION = "0.8.7".freeze
```

Comment:
Can you revert this please. We bump versions in separate commits / branches.

Reply:
I reverted and put the change log and version change in different commits.  Look good?

Reply:
Good catch, Patrick.